CHANGES.md

```diff
@@ -0,0 +1,31 @@
+### NUnit Xamarin Runner 3.6.0 - January 26, 2017
```

Comment:
3.6.0 was never released, so this should be changed to the current release 3.6.1 and recently fixed issues added.